libs/s25main/ingameWindows/iwOptionsWindow.cpp

```diff
@@ -41,65 +44,103 @@ enum
     ID_btMusicPlayer,
     ID_btAdvanced,
     ID_btSurrender,
-    ID_btEndGame
+    ID_btEndGame,
+    ID_cpBirdSounds
 };
-}
+
+constexpr auto windowSize = Extent(300, 545);
+constexpr auto centerPosition = windowSize.x / 2;
+constexpr auto rowHeight = 40;
+constexpr auto rowHeightSmall = 31;
+constexpr auto textSpacing = 17;
+constexpr auto textPadding = 35;
+constexpr auto headerStartPosition = DrawPoint(centerPosition, 10);
+} // namespace
 
 iwOptionsWindow::iwOptionsWindow(SoundManager& soundManager)
-    : IngameWindow(CGI_OPTIONSWINDOW, IngameWindow::posLastOrCenter, Extent(300, 515), _("Game menu"),
+    : IngameWindow(CGI_OPTIONSWINDOW, IngameWindow::posLastOrCenter, windowSize, _("Game menu"),
                    LOADER.GetImageN("resource", 41)),
       soundManager(soundManager)
 {
-    // Der Soldat oben
-    AddImage(ID_imgSoldier, DrawPoint(150, 36), LOADER.GetImageN("io", 30));
+    using Offset = DrawPoint;
+    constexpr Extent buttonImageSize(35, 35);
+    constexpr Extent optionSizeSmall(160, 22);
+    constexpr Extent optionSizeBig(168, 24);
+    constexpr Offset textOffset(50, 24);
+    constexpr Offset optionOffset(65, 6);
 
-    AddText(ID_txtRttr, DrawPoint(150, 60), "Return To The Roots", COLOR_YELLOW, FontStyle::CENTER, NormalFont);
-    AddText(ID_txtVersion, DrawPoint(150, 77), rttr::version::GetReadableVersion(), COLOR_YELLOW, FontStyle::CENTER,
-            NormalFont);
-    AddFormattedText(ID_txtCopyright, DrawPoint(150, 94),
+    DrawPoint curPos = headerStartPosition;
+
+    // The soldier on top
+    constexpr Offset soldierOffset(0, 26);
+    AddImage(ID_imgSoldier, curPos + soldierOffset, LOADER.GetImageN("io", 30));
+    curPos.y += textSpacing + soldierOffset.y;
+
+    AddText(ID_txtRttr, curPos, "Return To The Roots", COLOR_YELLOW, FontStyle::CENTER, NormalFont);
+    curPos.y += textSpacing;
+
+    AddText(ID_txtVersion, curPos, rttr::version::GetReadableVersion(), COLOR_YELLOW, FontStyle::CENTER, NormalFont);
+    curPos.y += textSpacing;
+
+    AddFormattedText(ID_txtCopyright, curPos,
                      "\xC2\xA9"
                      "2005 - %s Settlers Freaks",
                      COLOR_YELLOW, FontStyle::CENTER, NormalFont)
       % rttr::version::GetYear();
+    curPos.y += rowHeightSmall;
+    curPos.x = textPadding;
```

Comment:
Name seems of: It isn't used for text

```suggestion
    curPos.x = leftMargin;
```